kornia/augmentation/augmentation.py

```diff
@@ -58,7 +58,7 @@ def forward(self, input: Union[torch.Tensor, Tuple[torch.Tensor, torch.Tensor]],
             output = self.apply_transform(input[0], self._params)
             transformation_matrix = self.compute_transformation(input[0], self._params)
             if return_transform:
-                return output, input[1] @ transformation_matrix
+                return output, transformation_matrix @ input[1]
```

Comment:
can we add a small numeric test for this ? just to make sure that numbers are ok so that we can go forward and backward through the transforms chain

Reply:
Added a gradtest but I'm not sure if that's what you meant.

Reply:
that's okay too but was referring to something like in line 61 but with actual numbers - not random stuff since it a bit difficult to follow. In any case, seems that's in this way testing is robust, however, my concern is that you didn't break any other test with this change. We have to improve testing on whole augmentations. In fact coverage is starting to decrease /cc @shijianjian